plaso/formatters/setupapi.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+"""Windows Setupapi log event formatter."""
+
+from __future__ import unicode_literals
+
+from plaso.formatters import interface
+from plaso.formatters import manager
+
+
+class SetupapiLogFormatter(interface.ConditionalEventFormatter):
+  """Formatter for a Windows Setupapi log file event."""
+
+  DATA_TYPE = 'setupapi:log:line'
+
+  FORMAT_STRING_PIECES = [
+      'Description: {entry_type} |',
```

Comment:
If you want the values to be separated by ' | ' override FORMAT_STRING_SEPARATOR
Also see: https://github.com/log2timeline/plaso/blob/master/plaso/formatters/interface.py#L220